packager/packager.cc

```diff
@@ -88,16 +88,24 @@ struct PackageName {
     }
 };
 
-enum class ImportType {
+enum class ImportType : uint8_t {
```

Comment:
There's a handful of semi-related changes in this PR. Feel free to review by commit for a better breakdown.